addons/l10n_ar/tests/common.py

```diff
@@ -747,4 +747,3 @@ def _set_today_rate(self, currency, value):
             rate.rate = value
         else:
             rate_obj.create({'company_id': self.env.company.id, 'currency_id': currency.id, 'rate': value})
-
```

Comment:
pycharm doing his thing :eyes:

Reply:
You can just remove all diff on that file though

Reply:
i need to remove my automatic end of file thingy, i will do it

Reply:
You could also just have reset the file with git :wink: 
i.e. `git checkout HEAD~ -- addons/l10n_ar_tests/common.py`